exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/data_customer_support.ipynb

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
<p>Please clear the output from all cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>